osbs/build/build_request.py

```diff
@@ -562,6 +562,7 @@ def render_add_labels_in_dockerfile(self):
             'vendor': self.spec.vendor,
             'authoritative-source-url': self.spec.authoritative_registry,
             'distribution-scope': self.spec.distribution_scope,
+            'release': self.spec.release,
```

Comment:
@twaugh so what you said makes sense, you don't want to push 'release':None to the orchestrator.  However, isn't that already filtered by the code as it translates 'labels' into 'implicti_labels' as shown below?

        implicit_labels = {}
        label_spec = {
            'vendor': self.spec.vendor,
            'authoritative-source-url': self.spec.authoritative_registry,
            'distribution-scope': self.spec.distribution_scope,
            'release': self.spec.release,

>>     I blindly add 'release' above, which is the red flag you have
        }

        for label, spec in label_spec.items():
            if spec.value is not None:
                implicit_labels[label] = spec.value

>>> but the above for-loop checks for None and prevents it from being added to implicit_labels if it is None, correct?

        self.dj.dock_json_merge_arg(phase, plugin, 'labels', implicit_labels)

>>> the resulting filtered 'implicit_labels' is passed here. 
>>> So I think everything is fine, no?